k8s/examples/nginx/README.md

```diff
@@ -0,0 +1,50 @@
+# Nginx on Kubernetes with StorageOS persistent Storage
+
+This example shows an example of how to deploy Nginx on Kubernetes, in order to
+exfiltrate files that have been written to a StorageOS persistent volume. The
+files create a stateful set that can be used *AFTER* a StorageOS cluster has
+been created. For more information on how to install a StorageOS cluster please see
+[the StorageOS documentation](https://docs.storageos.com/docs/introduction/quickstart)
+for more information.
```

Comment:
For more information appears twice in the same sentence. With one I think is enough.